SpatialGDK/Extras/schema/worker_extents.schema

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+package unreal;
```

Comment:
Can you increment `RequiresSetup` to ensure this new schema file is copied correctly.